references/hw2.bib

```diff
@@ -0,0 +1,6 @@
+@book{fox,
```

Comment:
Great report!

I added minor comments. My main thoughts are around running site-specific models instead of a single model (with interactions) and the interpretation of model coefficients (absolute vs relative effects). Yet, I don't believe these points are critical.

I really liked the language which made the report accessible to a wide audience, the careful discussion about covariance structures, and the caution about relative measures of sugary/zero-cal sales.


Great job!